assets/sass/scaffolding/header.scss

```diff
@@ -235,16 +269,20 @@ $offscreenNavWidth: 65%;
         fill: palette('soft-grey');
     }
 
-    @include mq('tablet') {
+    @include mq(tablet) {
```

Comment:
The convention for this seems to be the other way around (69 matches for `mq('tablet')` versus 4 for `mq(tablet)` in my repo) – happy to use it unquoted if that's neater but can we make it one or the other?